packages/taiko-client/pkg/rpc/blob_datasource.go

```diff
@@ -115,7 +118,7 @@ func (ds *BlobDataSource) getBlobFromServer(ctx context.Context, blobHash common
 		err        error
 	)
 	if ds.socialScanEndpoint != nil {
-		route = "/blob/" + blobHash.String()
+		route = "/blobs/" + blobHash.String()
```

Comment:
During `hive-test` I found that calling `http://localhost:3001/blobs/xxx` is the right way to get blob content.